sql/moz-fx-data-shared-prod/firefox_ios/baseline_clients_last_seen_enriched/view.sql

```diff
@@ -0,0 +1,44 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_ios.baseline_clients_last_seen_enriched`
+AS
+-- This view is being added temporarily until issues preventing
+-- https://github.com/mozilla/bigquery-etl/pull/5434
+-- from merging have been resolved.
+SELECT
+  last_seen.*,
+  CASE
+    WHEN LOWER(isp) = 'browserstack'
+      THEN CONCAT("Firefox iOS", ' ', isp)
+    ELSE "Firefox iOS"
+  END AS app_name,
+  -- Activity fields to support metrics built on top of activity
+  CASE
+    WHEN BIT_COUNT(days_active_bits)
+      BETWEEN 1
+      AND 6
+      THEN 'infrequent_user'
+    WHEN BIT_COUNT(days_active_bits)
+      BETWEEN 7
+      AND 13
+      THEN 'casual_user'
+    WHEN BIT_COUNT(days_active_bits)
+      BETWEEN 14
+      AND 20
+      THEN 'regular_user'
+    WHEN BIT_COUNT(days_active_bits) >= 21
+      THEN 'core_user'
+    ELSE 'other'
```

Comment:
Something to note: `BIT_COUNT(days_active_bits)` can resolve to 0 if the user is sending pings but does not meet the criteria for qualified usage (i.e. `BIT_COUNT(days_seen_bits) > 0`). In this case those users will end up in the 'other' bucket. I think this is fine, actually, because (1) they are users that don't meet any of the activity criteria and (2) we don't use these activity fields very often. But it's something to keep in mind as we switch from using `days_seen_bits` to `days_active_bits` here -- we're going to see inflation in the "other" category that we wouldn't have seen before the change.

Reply:
This specific part has been proposed by @bani so I assume this is something she already thought about. The general idea here is that this should be flexible and this specific value will be calculated at runtime.

Reply:
I don't remember the conversation about activity segments and it isn't something I have worked with before, but I think classifying as other should work.

Reply:
hmm, ok. Apologies @bani, my memory must have let me down here. I recall us having conversations around keeping this in the views and not the queries, maybe that's why I thought you know more about them.

Reply:
Right, that was for lifecycle stage, which is kind of similar. Activity segment is also probably best left in a view as it's not super set in stone.